content/docs/command-reference/exp/show.md

```diff
@@ -78,36 +76,12 @@ metric or param.
 - `--param-deps` - include only parameters that are stage dependencies.
 
 - `--only-changed` - show only parameters and metrics with values that vary
-  across experiments. Note that this option takes precedence over
-  `--include-params` and `--include-metrics`, for example given
-  `--include-params=foo --only-changed`, param `foo` would still be hidden if
-  its value is the same in all experiments.
-
-- `--include-params <list>` - show the specified `dvc params` in the table only.
-  Accepts a comma-separated `list` of param names. Shell style wildcards
-  supported: `*`, `?`, `[seq]`, `[!seq]`, and `**` If a `path:` prefix is
-  included in the name, the wildcard pattern will **not** be matched against
-  that prefix but only against the rest of the name.
-
-- `--exclude-params <list>` - hide the specified `dvc params` from the table
-  (all param will be shown except for these). Accepts a comma-separated `list`
-  of param names. Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`,
-  and `**` If a `path:` prefix is included in the name, the wildcard pattern
-  will **not** be matched against that prefix but only against the rest of the
-  name.
-
-- `--include-metrics <list>` - show the specified `dvc metrics` in the table
-  only. Accepts a comma-separated `list` of metric names. Shell style wildcards
-  supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`. If a `path:` prefix is
-  included in the name, the wildcard pattern will **not** be matched against
-  that prefix but only against the rest of the name.
-
-- `--exclude-metrics <list>` - hide the specified `dvc metrics` from the table
-  (all param will be shown except for these). Accepts a comma-separated `list`
-  of metric names. Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`,
-  and `**`. If a `path:` prefix is included in the name, the wildcard pattern
-  will **not** be matched against that prefix but only against the rest of the
-  name.
+  across experiments.
+
+- `--drop <list>` - remove the columns matching the specified regex patterns.
```

Comment:
Does this accept a comma-separated list or just a single regex? Could you explain this a bit more and how to use in the basic case where regex isn't needed? Probably not all users will be familiar with regex.

Reply:
Does it make sense to keep supporting a list (comma separated `--drop 'x,y'` or multiple `--drop x --drop y`) now that it supports regex?

I understand that some users might not be familiar with regex but, if we show an example of how `--drop 'x|y'` would be equivalent as passing multiple values, I'm no longer sure if the lack of regex knowledge would be a problem at all.

Reply:
Could there be more than one `--drop` or `--keep` in a command? Like `dvc exp show --drop this --drop that`?

Reply:
Updated core to accept single value only and added example for `x|y`

Reply:
> Could there be more than one `--drop` or `--keep` in a command? Like `dvc exp show --drop this --drop that`?

No. I added an example on how `this|that` could achieve the same